<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectScoreManage</class>
 <widget class="QWidget" name="ProjectScoreManage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;成绩管理&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1.设置项目成绩</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ProjectPath">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>点击挑选项目所在的路径</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>2.查看现有项目及成绩</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>修改项目(删除,修改成绩)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ViewProjects">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>90</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>查看项目</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>3.项目排序及去重</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>140</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>项目排序及去重</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="SetProjectYorN">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>220</y>
     <width>166</width>
     <height>24</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="Widget1">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>401</width>
     <height>201</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>file(name)</string>
    </property>
    <property name="foreground">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>255</red>
       <green>170</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
   </column>
   <column>
    <property name="text">
     <string>type</string>
    </property>
    <property name="foreground">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>255</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
   </column>
   <column>
    <property name="text">
     <string>score</string>
    </property>
    <property name="foreground">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>255</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
   </column>
   <column>
    <property name="text">
     <string>path</string>
    </property>
    <property name="foreground">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>255</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:7pt;&quot;&gt;请输入需要删除的项目的行号&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="deleteproject">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>删除项目</string>
   </property>
  </widget>
  <widget class="QLabel" name="deletefalse_1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>输入非法!</string>
   </property>
  </widget>
  <widget class="QLabel" name="deleteproject_true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>删除成功!</string>
   </property>
  </widget>
  <widget class="QLabel" name="deletefalse_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>无此行!</string>
   </property>
  </widget>
  <widget class="QLabel" name="projectpaixu">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>270</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;项目排序去重成功&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ProjectPath</sender>
   <signal>clicked()</signal>
   <receiver>ProjectScoreManage</receiver>
   <slot>slot1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>414</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>419</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>ProjectScoreManage</receiver>
   <slot>deleteproject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>72</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SetProjectYorN</sender>
   <signal>accepted()</signal>
   <receiver>ProjectScoreManage</receiver>
   <slot>ProjectsPathAccept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SetProjectYorN</sender>
   <signal>rejected()</signal>
   <receiver>ProjectScoreManage</receiver>
   <slot>ProjectsPathReject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>378</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ViewProjects</sender>
   <signal>clicked()</signal>
   <receiver>ProjectScoreManage</receiver>
   <slot>viewprojects()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>465</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>ProjectScoreManage</receiver>
   <slot>selectproject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>404</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>516</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot1()</slot>
  <slot>viewprojects()</slot>
  <slot>selectproject()</slot>
  <slot>PushButtonClicked()</slot>
  <slot>PushButtonClicked2()</slot>
  <slot>slot2()</slot>
  <slot>ProjectsPathReject()</slot>
  <slot>ProjectsPathAccept()</slot>
  <slot>deleteproject()</slot>
  <slot>slot4()</slot>
 </slots>
</ui>
